### DISCUSSION TITLE
My wishlist

### DISCUSSION THREAD

**Discussion by crumblingstatue** [yds12/tarsila#8] [Ideas]
Don't know if any of these are implemented or planned, but here are some things I really want:

- [x] Zooming (integer granularity is fine)
- [ ] Configurable grid
- [ ] Crosshair cursor for better idea where you are going to put a pixel

**Comment by yds12** [comment]
Zooming does exist (the plus and minus keys, without pressing SHIFT). I plan to add zooming based on the mouse scroll, and also the possibility of just entering the zoom level to the exact desired percentage. Note that these keyboard shortcuts in the future will be all configurable.

Configurable grid is planned (do you mean the checkered background, or simply lines that will appear every N pixels?).

The crosshair cursor is a great idea, thanks! This will need some design decisions: should we use it only for brush, for all tools, or should it be something you can toggle on and off?

This is a [features wishlist](https://github.com/yds12/tarsila/blob/master/docs/features.md) I was using as a roadmap: (I will be moving this list to another place soon -- probably to the [roadmap](https://github.com/yds12/tarsila/blob/master/ROADMAP.md)). There is also this [list of known bugs/issues](https://github.com/yds12/tarsila/blob/master/docs/issues.md) that will be transferred to the issues page.

**Reply by crumblingstatue** [reply]
> Configurable grid is planned (do you mean the checkered background, or simply lines that will appear every N pixels?).

Lines that appear every N image pixels

> The crosshair cursor is a great idea, thanks! This will need some design decisions: should we use it only for brush, for all tools, or should it be something you can toggle on and off?

It would ideally be configurable.

**Reply by yds12** [reply]
I created this issue for the crosshair cursor: https://github.com/yds12/tarsila/issues/11
